results_spreadsheet.py

```diff
@@ -0,0 +1,136 @@
+import os
+import pandas as pd
+from tqdm import tqdm
+
+# Paths
+base_path = "./results/fy25"
+summary_file = "./analysis/results_list.csv"
+output_excel = "./analysis/results_spreadsheet.xlsx"
+mapping_file = "./data/county2zone.csv"
+
+# Technologies to collapse
+agg_techs = ["wind-ons", "wind-ofs", "pv", "csp", "hyd", "egs", "coal", "gas"]
+
+
+# === Helpers ===
+def load_region_map(path):
+    df = pd.read_csv(path)
+    if "ba" in df.columns and "r" not in df.columns:
+        df = df.rename(columns={"ba": "r"})
+    return df.set_index("r")["state"].to_dict()
+
+
+def clean_and_aggregate(df, scenario, region_to_state):
+    # Normalize headers
+    df.columns = df.columns.str.strip().str.lower()
+
+    # Map either 'r' or 'ba' to state (whichever exists)
+    col = {"r", "ba"}.intersection(df.columns)
+    if col:
+        col = col.pop()
+        df["state"] = df[col].map(region_to_state)
+        df = df.drop(columns=[col])
+
+    # collapse i
+    if "i" in df.columns:
+        df["i"] = df["i"].astype(str).str.strip().str.lower()
+        for tech in agg_techs:
+            if tech in ["coal", "gas"]:
+                df.loc[
+                    df["i"].str.contains(tech, case=False, na=False)
+                    & ~df["i"].str.contains("ccs", case=False, na=False),
+                    "i",
+                ] = tech
+                df.loc[
+                    df["i"].str.contains(tech, case=False, na=False)
+                    & df["i"].str.contains("ccs", case=False, na=False),
+                    "i",
+                ] = f"{tech}_ccs"
+            else:
+                df.loc[df["i"].str.contains(tech, case=False, na=False), "i"] = tech
+
+    # add Scenario
+    df.insert(0, "scenario", scenario)
+
+    # aggregate value
+    # Identify numeric columns to aggregate
+    if "value" in df.columns:
+        agg_cols = ["value"]
+    elif {"tons", "md", "damage_$", "mortality"}.issubset(df.columns):
+        agg_cols = ["tons", "md", "damage_$", "mortality"]
```

Comment:
I was confused for a bit, here because I intuit `agg_cols` to have a different meaning than is intended. But OK. My other comment is that this violates the D.R.Y. principle (Don't Repeat Yourself). Let's do the following instead.

```suggestion
    # aggregate value
    # Identify numeric columns to aggregate
    health_cols = ["tons", "md", "damage_$", "mortality"]
    
    if "value" in df.columns:
        agg_cols = ["value"]
    elif set(health_cols).issubset(df.columns):
        agg_cols = health_cols